float64[] initialState
float64 x
float64 y
float64 z
float64[] rotx
float64[] roty
float64[] rotz
---
float64[] solution
int32 error
